packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -473,16 +473,15 @@ function hasAnyConfiguration(provider: ProviderInfo) {
                   <ConnectionErrorInfoButton status="{status}" />
                 {/if}
               </div>
-              <div class="mt-2">
-                <div class="text-gray-700 text-xs" aria-label="{container.name} type">
-                  {#if container.type === 'docker'}Docker{:else if container.type === 'podman'}Podman{/if} endpoint
-                </div>
-                <div class="mt-1">
-                  <span
-                    class="my-auto text-xs"
-                    class:text-gray-900="{container.status !== 'started'}"
-                    aria-label="{container.name} endpoint">{container.endpoint.socketPath}</span>
-                </div>
+              <div class="mt-2 text-gray-700 text-xs" aria-label="{container.name} type">
+                {#if container.type === 'docker'}Docker{:else if container.type === 'podman'}Podman{/if} endpoint
+              </div>
+              <div
```

Comment:
One alternative is to remove the socketPath from the summary, and only show the type.

Reply:
could be a follow-up PR/improvement